app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -461,6 +463,7 @@ ShortFormApplicationController = (
   $scope.$on '$stateChangeSuccess', (e, toState, toParams, fromState, fromParams) ->
     $scope.handleErrorState() if $state.params.error
     ShortFormNavigationService.isLoading(false)
+    # if I've reached the confirmation page and I'm logged in, call Idle.unwatch()
```

Comment:
I should have removed that note... I implemented it in lines 442-446 by checking to see if the user is logged in before throwing the alert.